libsolidity/interface/StandardCompiler.cpp

```diff
@@ -657,72 +657,82 @@ std::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompiler:
 
 	ret.errors = Json::arrayValue;
 
-	for (auto const& sourceName: sources.getMemberNames())
+	if (ret.language != "SolidityAST")
```

Comment:
maybe better to check for `ret.language == "Solidity"` here

Reply:
Yes, sounds better - you can add an ultimate ``else`` clause with ``astAssert(false);`` or so in the end (assuming that we already validated that no other languages occur up front)